guides/source/asset_pipeline.md

```diff
@@ -868,218 +779,384 @@ each asset will have a different file name when it is changed. This way you
+developers seeking flexibility and performance.
+
+#### How `jsbundling-rails` Works
+
+1. After installation, it sets up your Rails app to use your chosen JavaScript
+   bundler.
+2. It creates a `build` script in your `package.json` file to compile your
+   JavaScript assets.
+3. During development, the `build:watch` script ensures live updates to your
+   assets as you make changes.
+4. In production, the gem ensures that JavaScript is built and included during
+   the precompilation step, reducing manual intervention. It hooks into Rails'
+   `assets:precompile` task to build JavaScript for all entry points during
+   deployment. This integration ensures that your JavaScript is production-ready
+   with minimal configuration.
+
+The gem automatically handles entry-point discovery - identifying the primary
+JavaScript files to bundle by following Rails conventions, typically looking in
+directories like `app/javascript/` and configuration. By adhering to Rails
+conventions, `jsbundling-rails` simplifies the process of integrating complex
+JavaScript workflows into Rails projects.
+
+#### When Should You Use It?
+
+`jsbundling-rails` is ideal for Rails applications that:
+
+- Require modern JavaScript features like ES6+, TypeScript, or JSX.
+- Need to leverage bundler-specific optimizations like tree-shaking, code
+  splitting, or minification.
+- Use `Propshaft` for asset management and need a reliable way to integrate
+  precompiled JavaScript with the broader Rails asset pipeline.
+- Utilize libraries or frameworks that depend on a build step. For example,
+  projects requiring transpilation—such as those using
+  [Babel](https://babeljs.io/), [TypeScript](https://www.typescriptlang.org/),
+  or React JSX—benefit greatly from `jsbundling-rails`. These tools rely on a
+  build step, which the gem seamlessly supports.
+
+By integrating with Rails tools like `Propshaft` and simplifying JavaScript
+workflows, `jsbundling-rails` allows you to build rich, dynamic front-ends while
+staying productive and adhering to Rails conventions.
+
+### `cssbundling-rails`
+
+[`cssbundling-rails`](https://github.com/rails/cssbundling-rails) integrates
+modern CSS frameworks and tools into your Rails application. It allows you to
+bundle and process your stylesheets. Once processed, the resulting CSS is
+delivered via the Rails asset pipeline.
+
+#### How `cssbundling-rails` Works
+
+1. After installation, it sets up your Rails app to use your chosen CSS
+   framework or processor.
+2. It creates a `build:css` script in your `package.json` file to compile your
+   stylesheets.
+3. During development, a `build:css --watch` task ensures live updates to your
+   CSS as you make changes, providing a smooth and responsive workflow.
+4. In production, the gem ensures your stylesheets are compiled and ready for
+   deployment. During the `assets:precompile` step, it installs all
+   `package.json` dependencies via `yarn` and runs the `yarn build:css` command
```

Comment:
```suggestion
   `package.json` dependencies via `yarn` or `bun` and runs the `yarn build:css`
   or `bun build:css` command
```

To my understanding, `bun` [can also be used](https://github.com/rails/cssbundling-rails/blob/123569431c0e303c1aca8b275b2d1bc3c68a3a63/lib/install/helpers.rb#L5), even if this is not mentioned in the cssbundling-rails [README](https://github.com/rails/cssbundling-rails).